service/__init__.py

```diff
@@ -4,17 +4,28 @@
 import dateutil
 import dateutil.parser
 import json
+from flask_login import LoginManager
+from flask_wtf.csrf import CsrfProtect
 
 from config import CONFIG_DICT
 
 
 app = Flask(__name__)
 app.config.update(CONFIG_DICT)
+app.secret_key = 'change_this_key-not_secure'
+
+login_manager = LoginManager()
+login_manager.init_app(app)
+login_manager.login_view='/login'
+
+# csrf = CsrfProtect()
```

Comment:
It would be best to remove these lines and create an item that would cover this problem (in Trello).